claims.go

```diff
@@ -90,15 +90,24 @@ func (c *StandardClaims) VerifyNotBefore(cmp int64, req bool) bool {
 
 // ----- helpers
 
-func verifyAud(aud string, cmp string, required bool) bool {
-	if aud == "" {
+func verifyAud(aud []string, cmp string, required bool) bool {
+	if len(aud) == 0 {
 		return !required
 	}
-	if subtle.ConstantTimeCompare([]byte(aud), []byte(cmp)) != 0 {
-		return true
-	} else {
-		return false
+	var stringClaims string
+	for _, a := range aud {
+		if subtle.ConstantTimeCompare([]byte(a), []byte(cmp)) != 0 {
+			return true
```

Comment:
Just a minor thing, but this probably breaks the constant time checking, because it immediately returns once one audience was correct vs. checking all audience whether they are correct or not in constant time. 

Not sure, if constant time is really needed for the audience, through.

Reply:
good spot, how about checking everything and using a var to store if any matched?

Reply:
changed to use a var and always check all items in the slice